packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -116,50 +106,104 @@ function FullWidthAlert(props: FullWidthAlertProps) {
-  return (
+
+  const gridSize = 12
+  const buttonGridSize = 4
+  const hasButtons = primaryButtonConfig || secondaryButtonConfig
+  const buttonGridSizeMap = {
+    sm: hasButtons ? buttonGridSize + 1 : 0,
+    md: hasButtons ? buttonGridSize - 1 : 0,
+    lg: hasButtons ? buttonGridSize : 0,
+  }
+
+  const alert = (
     <Alert
-      variant={variant}
-      show={show}
-      dismissible={false}
-      transition={transition}
-      className={`FullWidthAlert bootstrap-4-backport ${
-        isGlobal ? 'global' : ''
-      } ${additionalAlertVariantClass}`}
+      severity={variantToSeverity(variant)}
+      sx={{
+        width: '100%',
+        '.MuiAlert-message': {
+          flexGrow: 1,
+        },
+      }}
+      className={`FullWidthAlert ${isGlobal && 'global'}`}
+      onClose={onClose}
     >
-      <div className={`gridContainer ${onClose ? 'hasCloseButton' : ''}`}>
-        <span className="iconArea">{iconContent}</span>
-        <span className="mainContent">
-          <span className="messageArea">
-            <Typography variant="smallText2">{title}</Typography>
-            {title && description && <div className="SRC-marginBottomTen" />}
-            <Typography variant="smallText1">{description}</Typography>
-          </span>
-          <span className="buttonArea">
-            {secondaryButtonConfig && (
-              <ButtonFromConfig
-                className="secondaryButton"
-                config={secondaryButtonConfig}
-                variant="tertiary"
-              />
-            )}
-            {primaryButtonConfig && (
-              <ButtonFromConfig
-                className="primaryButton"
-                config={primaryButtonConfig}
-                variant="sds-primary"
-              />
-            )}
-          </span>
-        </span>
-        {onClose && (
-          <button className="closeAlert" onClick={onClose}>
-            <Clear fontSize={'large'} />
-          </button>
+      <Grid
+        container
+        rowSpacing={1}
+        columnSpacing={2}
+        direction={{ xs: 'column', sm: 'row' }}
+        alignItems={{ xs: 'start', sm: 'center' }}
+      >
+        <Grid
+          item
+          xs={gridSize}
+          sm={gridSize - buttonGridSizeMap.sm}
+          md={gridSize - buttonGridSizeMap.md}
+          lg={gridSize - buttonGridSizeMap.lg}
+        >
+          {title && <AlertTitle>{title}</AlertTitle>}
+          {description}
+        </Grid>
+        {hasButtons && (
+          <Grid
+            item
+            xs={gridSize}
+            sm={buttonGridSizeMap.sm}
+            md={buttonGridSizeMap.md}
+            lg={buttonGridSizeMap.lg}
+            display="flex"
+            justifyContent="flex-end"
+          >
+            <Stack
+              spacing={{ xs: 1, lg: 2 }}
+              direction={{
+                xs: 'column-reverse',
+                sm: 'column',
+                lg: 'row',
+              }}
+            >
```

Comment:
Adjust buttons alignment based on screen size, so that they are visible and match the [responsive button positioning](https://www.figma.com/file/bALXz8AOxdzQyrsSd4RkVm/Governance-UI%3A-Registration-and-Data-Access?type=design&node-id=2579-11907&t=D8h9BkAgQFX7QOr9-4) in the [Welcome/Getting Started modules](https://sagebionetworks.jira.com/browse/PORTALS-2470) -- 
  - When screen is extra small, buttons will be stacked beneath text, so provide full width.
  - When screen is small, buttons will still be side-by-side with text, but overall space will be smaller, so provide more space to buttons so that text doesn't wrap within buttons. 
  - When screen is medium, buttons will be stacked, but side-by-side with text, so provide less space. 
  - When screen is large, buttons will be side-by-side. 

screen size | main | feature
:---:|:---:|:---:
Small mobile | <img width="356" alt="main_FullWidthAlert_SmallMobile" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/3ebbaa18-3713-4846-8e33-dea1f9a7efae"> | <img width="371" alt="feature_FullWidthAlert_SmallMobile" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/33a960a9-436f-46f2-8661-9bd0a35f9ad3">
Large mobile | <img width="446" alt="main_FullWidthAlert_LargeMobile" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/0f4dc30c-1059-49cd-80f5-9dadb96cc716"> | <img width="477" alt="feature_FullWidthAlert_LargeMobile" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/dccd4951-0f8f-4360-931e-0599111e57fa">